src/main/scala/com/typesafe/conductr/sandbox/sbt/ConductRSandbox.scala

```diff
@@ -215,12 +236,9 @@ object ConductRSandbox extends AutoPlugin {
 
     val logLevelEnv = Map("AKKA_LOGLEVEL" -> logLevelValue)
     val syslogEnv = cond0Ip.map(ip => Map("SYSLOG_IP" -> ip)).getOrElse(Map.empty)
-    val conductrFeaturesArgs =
-      if (featureNames.isEmpty)
-        Map.empty
-      else
-        Map("CONDUCTR_FEATURES" -> featureNames.mkString(","))
-    val envsArgs = (logLevelEnv ++ syslogEnv ++ envsValue ++ conductrFeaturesArgs).flatMap { case (k, v) => Seq("-e", s"$k=$v") }.toSeq
+    val conductrFeatureArgs = featureNames.toMapWithKey("CONDUCTR_FEATURES")(_.mkString(","))
```

Comment:
This is not really different to the above beside:
- Doing the check with `nonEmpty` instead of `isEmpty`
- Using `reduce` instead of `mkString`. `mkString` is more idiomatic and specific for this scenario
- Shrinking it to one line

With this PR we need to do the logic twice, therefor I think we should encapsulate this logic to a method. The new implicit method in the package object is very flexible by using a function for the transformation. We could e.g. use `mkString` and `reduce` with the new method. Also if you call the method you only need to specify the essentials (key name and the transformation). That what a higher-order function is about.

If you like I can change the logic from `isEmpty` to `nonEmpty`. I guess that is more idiomatic. WDYT?